source/stylesheets/site.css.scss

```diff
@@ -1,9 +1,13 @@
 @import "bourbon/bourbon";
-@import "base/variables";
-@import "base/typography";
-@import "base/mixins";
-@import "base/layout";
+@import "settings/variables";
+@import "settings/typography";
+
+@import "tools/mixins";
+
+@import "elements/layout";
+
 @import "components/navbar";
 @import "components/footer";
 @import "components/index";
 @import "components/about";
+@import "components/work";
```

Comment:
I like the organization that's happening here. Couple of suggestions to take it even further:

Separate bourbon from variables and typography, as bourbon is vendor-provided code, while the other two are logic you've authored for this project. There may be a point where you could be importing other vendor libraries, and/or writing additional logic, so I'd suggest pre-optimizing this now just in case. 

A short code comment before each section would also be good, just to be explicit about intent.

I'd also alphabetize the order of your components. It'll help with maintenance as the project grows.

Reply:
Sounds good - yeah I am attempting to follow the ITCSS architecture that was recommended to me https://www.xfive.co/blog/itcss-scalable-maintainable-css-architecture/

Reply:
I love Harry's work!